subjects/countstars/README.md

```diff
@@ -0,0 +1,43 @@
+## CountStars
+
+### Instructions
+Write a function called CountStars that counts the stars up to a number given as an argument.
+- If the number is negative or equal to 0, return "`No stars`"
+- No need to manage overflow.
+
+The Library strconv is allowed.
```

Comment:
remove this, this is never added in the readme it is in the allowed func.
If i ask for a comment on the allowed funcs you should just add a comment here on github. 
Also this package is not needed, they can create their own Itoa func as they have already done it in the piscine.

Reply:
this could be a level 3 or 4 exercise, if they use the strconv its a level 2 lets not make it that easy